Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN pip install redisvl==0.0.7 --no-deps
 # ensure pyjwt is used, not jwt
 RUN pip uninstall jwt -y
 RUN pip uninstall PyJWT -y
-RUN pip install PyJWT --no-cache-dir
+RUN pip install PyJWT==2.9.0 --no-cache-dir
```

Comment:
@Jacobh2 this would be out of sync with requirements.txt which sets it to 2.8.0 

can you also bump requirements.txt

Reply:
Done! 👍